drag.cc

```diff
@@ -119,10 +119,7 @@ Dragging_info::Dragging_info(
  *  Delete dragging info.
  */
 
-Dragging_info::~Dragging_info(
-) {
-	delete save;
-}
+Dragging_info::~Dragging_info() = default;
```

Comment:
Might as well remove it and remove the constructor declaration from the class definition.

Reply:
As you prefer. Personally I like to keep it explicit (in the header at least).